core/utils.py

```diff
@@ -97,3 +115,173 @@ def seconds2hms(seconds):
 00:20.000 --> 00:20.900
 Maybe they'll come closer.
 """
+
+
+class VTTCue:
```

Comment:
I love that you put all of this in a class. It's long enough that it might do better in its own file? I'm happy either way.